### ISSUE TITLE
Get "Unauthorized" in all router with middleware

### ISSUE THREAD

**Issue by gemins** [tymondesigns/jwt-auth#1242]
I'm using Tymon/Jwt-auth : 1.0@dev and Lumen 5.4.

After update my lumen from 5.4.19 to 5.4.27 was recive this message in all routes when is protected by middleware with JWT.

I'm using native AuthServiceProvider:
```
$this->app['auth']->viaRequest('api', function ($request){
            if($request->input('dni'))
                return User::where('dni', $request->input('dni'))->first();
            else if ($request->input('username'))
                return User::where('dni', $request->input('username'))->first();
            else if ($request->input('email'))
                return User::where('dni', $request->input('email'))->first();
}
```

In bootstrap/app.php
```
$app->routeMiddleware([
    'auth' => App\Http\Middleware\Authenticate::class
]);
$app->register(App\Providers\AppServiceProvider::class);
$app->register(App\Providers\AuthServiceProvider::class);
//$app->register(App\Providers\EventServiceProvider::class);
$app->register(Tymon\JWTAuth\Providers\LumenServiceProvider::class);
```

Before update my lumen, i was login with the system normaly and haven't problem. Now can login and get the Token, but when try to enter in a route with middleware i get the response: "Unauthorized"

Thanks you.

**Comment by sandbox-services** [comment]
confirmed: your lumen guide fails unless you set your lumen version to 5.4.1 (5.4.2->5.4.6) all return Unauthorized regardless if you're sending the jwt token via Authorize headers or token in query string

**Comment by tymondesigns** [comment]
closed by #1243
